isamples_metadata/taxonomy/metadata_models.py

```diff
@@ -108,18 +108,23 @@ def check_informative(self, text: str, description_map: dict) -> bool:
         # if record does not have description field
         #   && no CV words in content
         #       && only contains sampleType
-        informative = False
-        if "description" not in description_map:
-            for cv in SESARClassifierInput.SESAR_CV_words:
-                if cv in text:
-                    informative = True
-                    break
-        if not informative and "sampleType" in description_map and \
-                text == description_map["sampleType"]:
-            informative = False
-        else:
-            informative = True
-        return informative
+        # check descriptive
```

Comment:
You don't have to fix this but you could do something like 

```
if description_map.get("description", "") == "":
```

to make it more concise.

Reply:
The second argument to `get` is the default value that's returned if the value isn't contained in the map.

Reply:
https://docs.python.org/3/library/stdtypes.html#mapping-types-dict

```
get(key[, default])
Return the value for key if key is in the dictionary, else default. If default is not given, it defaults to None, so that this method never raises a [KeyError](https://docs.python.org/3/library/exceptions.html#KeyError).
```

Reply:
Thanks for the suggestion Danny!